tasks/release.py

```diff
@@ -906,3 +906,39 @@ def chase_release_managers(_, version):
     client = WebClient(os.environ["SLACK_API_TOKEN"])
     for channel in channels:
         client.chat_postMessage(channel=channel, text=message)
+
+
+@task
+def check_for_changes(ctx, release_branch, warning_mode=False):
+    """
+    Check if there was any modification on the release repositories since last release candidate.
+    """
+    next_version = next_rc_version(ctx, "7")
+    repo_data = generate_repo_data(warning_mode, next_version, release_branch)
+    changes = 'false'
```

Comment:
Why not a boolean?

Reply:
because it feeds an env variable and I was not sure how `True/False` would be interpreted as output (whereas a string I know)